src/app/page.tsx

```diff
@@ -249,49 +269,60 @@ export default function Home() {
       .catch(console.error);
   };
 
-  const [showSettings, setShowSettings] = useState<boolean>(false);
-  const [settings, setSettings] = useState({
-    radiancePath: "/usr/local/radiance/bin/",
-    hdrgenPath: "/usr/local/bin/",
-    outputPath: "/home/hdri-app/",
-    tempPath: "/tmp/",
-  });
-
-  const handleSettingsChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    const updatedSettings = JSON.parse(JSON.stringify(settings));
-    updatedSettings[event.currentTarget.name] = event.currentTarget.value;
-    setSettings(updatedSettings);
-  };
-
   return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24">
-      <div className="w-full">
-        <button
-          className="bg-gray-300 hover:bg-gray-400 text-gray-700 font-semibold py-1 px-2 border-gray-400 rounded"
-          onClick={() => setShowSettings((prev) => !prev)}
-        >
-          Settings
-        </button>
-      </div>
-
-      {showSettings ? (
-        <SettingsModal
-          settings={settings}
-          handleChange={handleSettingsChange}
-          toggleModal={() => setShowSettings((prev) => !prev)}
-        />
-      ) : (
-        <></>
-      )}
-
-      <h1>HDRICalibrationTool</h1>
+    <main className="bg-white flex min-h-screen flex-col items-center justify-between">
       <div>
-        <h2>Image Upload</h2>
+        <nav className="pt-10 bg-gray-300 fixed left-0 w-1/4 h-full">
+          <ul>
+            <li className="font-bold pt-5 pl-5">Navigation Configuration</li>
+            <li className="pt-5 pl-5">
+              <a href="#image_selection">Image Selection</a>
+            </li>
+            <li className="pt-5 pl-5">
+              <a href="#response">Response File</a>
+            </li>
+            <li className="pt-5 pl-5">
+              <a href="#c_r_v">Cropping, Resizing, and View Settings</a>
+            </li>
+            <li className="pt-5 pl-5">
+              <a href="#v">Vignetting Correction</a>
+            </li>
+            <li className="pt-5 pl-5">
+              <a href="#nd">Neutral Density Correction</a>
+            </li>
+            <li className="pt-5 pl-5">
+              <a href="#cf">Calibration Factor Correction</a>
+            </li>
+            <li className="pt-10 pl-5">
+              <button
+                className="bg-gray-700 hover:bg-gray-400 text-gray-300 font-semibold py-1 px-2 border-gray-400 rounded"
+                onClick={() => setShowSettings(!showSettings)}
+                >
+                Settings
```

Comment:
Perhaps we should put the settings button at the bottom of the side nav, so it's a little separated from the configuration navigation part of the side panel? Not sure, just a thought. I'm open to whatever everyone thinks is best!